exercises/practice/strain/.meta/Example.roc

```diff
@@ -0,0 +1,28 @@
+module [keep, discard]
+
+keep : List a, (a -> Bool) -> List a
+keep = \list, predicate ->
+    loop = \sublist, selected ->
+        when sublist is
+            [] -> selected
+            [first, .. as rest] ->
+                if predicate first then
+                    rest |> loop (List.append selected first)
+                else
+                    rest |> loop selected
+
+    loop list []
+
+discard : List a, (a -> Bool) -> List a
+discard = \list, predicate ->
+    loop = \sublist, rejected ->
```

Comment:
The name rejected doesn't really make sense here given that these are the elements that are being kept.

Reply:
Oh good catch, thanks!